.github/workflows/build.yml

```diff
@@ -193,8 +193,9 @@ jobs:
           if-no-files-found: ignore
 
   update-test-catalog:
+    name: Update Test Catalog
```

Comment:
there is a step having same naming. Maybe we can rename the step to `Push Test Catalog`?